packages/json-parser-products/test/mappings.spec.js

```diff
@@ -0,0 +1,620 @@
+import { oneLineTrim } from 'common-tags'
+import ProductMapping from '../src/mappings'
+
+describe('::ProductMapping', () => {
+  let productMapping
+  beforeEach(() => {
+    productMapping = new ProductMapping()
+  })
+  it('should be defined', () => {
+    expect(ProductMapping).toBeDefined()
```

Comment:
Is this test needed? If ProductMapping is not defined, the `beforeEach` will fail first

Reply:
TDD... Will be removed